City-Bureau/city-scrapers-spokane
# 🏗️ Build spider: Central Valley School District Board

## What's this PR do?
Adds a spider to scrape meetings from the boarddocs website for Central Valley School District Board. The new spider is called `spo_cvsd`.

## Why are we doing this?
Requested by City Bureau's partners in Spokane.

## Steps to manually test

1. Ensure the project is installed:
```
pipenv sync --dev
```
2. Activate the virtual env and enter the pipenv shell:
```
pipenv shell
```
3. Run the spider:
```
scrapy crawl spo_cvsd -O test_output.csv
```
4. Monitor the output and ensure no errors are raised.

5. Inspect `test_output.csv` to ensure the data looks valid. Target website is [here](https://go.boarddocs.com/wa/cvsd/board.nsf/public).

## Are there any smells or added technical debt to note? 
- Fixture set up in the testing suite for this spider is a little more complicated because we need to ensure `meta` is available in response. A comment has been left in the file to provide clarity.
- Just a note: `scrapy validate spo_cvsd` passes with 100% for all items for me.
- Not a big deal but the branch name for this repo is misspelled. Should be "build-cvsd". Just noting in case the reviewer fetches the branch and is confused.
